src/store/assets/assets-state.ts

```diff
@@ -1,8 +1,14 @@
-import { AssetsInterface } from '../../interfaces/assets.interface';
+import { BalanceInterface } from '../../interfaces/balance.interface';
 import { LoadableEntityState } from '../types';
 
+export interface TezosState {
+  balance?: string;
+  error?: string;
+}
+
 export interface AssetsState {
-  tokens: LoadableEntityState<AssetsInterface[]>;
+  tokens: LoadableEntityState<BalanceInterface[]>;
+  tezos?: TezosState;
```

Comment:
Why don't you handle `isLoading` state in `TezosState`?  
Maybe use 
```
tezos?: LoadableEntityState<string>;
```
instead?

Reply:
And set `tezosInitialValue` to omit this `undefined`s (`state?.tezos?.balance`)